README.md

```diff
@@ -1,41 +1,41 @@
-# Title
-
-GUDID Pipeline
+# gudid-data
 
 ### Team Ownership - Product Owner
-
-Team Ownership - Product Owner
+Data Team
 
 ## Getting Started
 
-Required for running this:
+Follow these instructions to generate a gudid dataset:
 
-1. Download and install Java 23 (Azul)
+1. Clone the [gudid-data repository](https://github.com/ikmdev/gudid-data)
 
-## Building and Running
+```bash
+git clone [Rep URL]
+```
 
-Follow the steps below to build and run on your local machine:
+2. Change local directory to `gudid-data`
 
-1. Clone the repository from GitHub to your local machine
+3. Download AccessGUDID_Delimited_Full_Release File from SNOMED CT/ACCESS GUDID: (https://www.nlm.nih.gov/healthit/snomedct/index.html / https://accessgudid.nlm.nih.gov)
```

Comment:
These are the download links to include:
- AccessGUDID: https://accessgudid.nlm.nih.gov/download
- FDA Product Code Classification Files: https://www.fda.gov/medical-devices/classify-your-medical-device/download-product-code-classification-files